dispatchers.ts

```diff
@@ -13,12 +13,19 @@ export class CommandDispatcher implements ICommandDispatcher {
 		private $commandsService: ICommandsService,
 		private $staticConfig: Config.IStaticConfig,
 		private $sysInfo: ISysInfo,
-		private $options: IOptions) { }
+		private $options: IOptions,
+		private $fs: IFileSystem) { }
 
 	public dispatchCommand(): IFuture<void> {
 		return(() => {
 			if (this.$options.version) {
-				this.$logger.out(this.$staticConfig.version);
+				let version = this.$staticConfig.version;
+
+				let json = this.$fs.readJson(this.$staticConfig.PATH_TO_PACKAGE_JSON).wait();
```

Comment:
You may require package.json, which has been already cached by $staticConfig.version but not a merge stopper.